dsl/src/test/scala/ru/itclover/tsp/dsl/v2/FunctionRegistryTest.scala

```diff
@@ -1,97 +1,183 @@
 package ru.itclover.tsp.dsl.v2
+
 import org.scalactic.{Equality, TolerantNumerics}
 import org.scalatest.{FlatSpec, Matchers}
 import org.scalatest.prop.PropertyChecks
 import ru.itclover.tsp.core.Result
 
 class FunctionRegistryTest extends FlatSpec with Matchers with PropertyChecks {
-  val funReg = DefaultFunctionRegistry
+  val funReg: DefaultFunctionRegistry.type = DefaultFunctionRegistry
```

Comment:
As far as I know, one can write just `val funReg: FunctionRegistry = DefaultFunctionRegistry`

Reply:
no, in that case compiler can't find case class for function registry